scripts/battery_percentage.sh

```diff
@@ -23,6 +25,6 @@ print_battery_percentage() {
 }
 
 main() {
-	print_battery_percentage
+    print_battery_percentage
```

Comment:
I should make this more clear, but we use tabs for indentation instead of four spaces.